space_robots/README.md

```diff
@@ -2,103 +2,112 @@
 
 The Space ROS Demo docker image uses the Space ROS docker image (*openrobotics/spaceros:latest*) as its base image. The Dockerfile installs all of the prerequisite system dependencies along with the demo source code, then builds the Space ROS Demo.
 
+This is for Curiosity Mars rover and Canadarm demos.
+
 ## Building the Demo Docker
 
 To build the docker image, run:
 
 ```
-$ ./build.sh
+./build.sh
 ```
 
 ## Running the Demo Docker
 
 run the following to allow GUI passthrough:
 ```
-$ xhost +local:docker
+xhost +local:docker
 ```
 
 Then run:
 ```
-$ ./run.sh
+./run.sh
```

Comment:
Whether the `$` is included or not should be consistent across all our documentation. Is there a style guide on this for ROS 2 documentation? We should do whatever is done in that repository.

Reply:
Reverted